### ISSUE TITLE
prune output too verbose with the addition of archive id

### ISSUE THREAD

**Issue by FabioPedretti** [borgbackup/borg#3238]
As a follow-up to #3111, since 1.1 borg prune also adds the archive id when listing archives. This is a bit annoying when sending borg prune output via mail, since the mail client may split lines.
It would be nice to have a way to avoid printing the archive id, I don't know how that can be useful to have it by default on both prune and list (maybe also omits it by default on both).
This is just a minor issue, feel free to close as won't fix, I can easily send prune output to `| cut -c1-61` to do what I want, but maybe it can also be useful to others.
Thanks for all the work on borg!

**Comment by ThomasWaldmann** [comment]
Maybe we can have `--format` there.

**Comment by ThomasWaldmann** [comment]
Continue after #3235 is merged.

**Comment by jdchristensen** [comment]
While discussing the prune output, I'd like the option to only show the archives that are pruned, not those that are kept.  (For me, only one or two are pruned each time, but dozens are kept, which clutters the output.)
